recipes-support/snapd/snapd-go.inc

```diff
@@ -0,0 +1,70 @@
+inherit go-mod
+
+GO_IMPORT = "github.com/snapcore/snapd"
+
+# disable shared runtime for x86
+# https://forum.snapcraft.io/t/yocto-rocko-core-snap-panic/3261
+# GO_DYNLINK is set with arch overrides in goarch.bbclass
+GO_DYNLINK:x86 = ""
+GO_DYNLINK:x86-64 = ""
+GO_DYNLINK:arm = ""
+GO_DYNLINK:aarch64 = ""
+# modules are vendored in the release tarball
+GOBUILDFLAGS:append = " -mod=vendor"
+
+# tags to disable unnecessary functionality:
+# nosecboot  - disable secboot support, only useful for Ubuntu Core
+# nomanagers - disable the managers code which could have been be imported but
+#              isn't useful in binaries that aren't snapd
```

Comment:
This is confusing, we _are_ building snapd. So what is being disabled here? (I know but the comment is not clear IMO)